### DISCUSSION TITLE
Is there a way to reorder how comments are listed in issues?

### DISCUSSION THREAD

**Discussion by frodera** [community/community#33387] [Projects and Issues]
Can you order the comments *from newest to oldest* on the issue and project views? It would be a very handy feature for issues with many comments.

**Comment by itskemo** [comment]
Unfortunately it is not possible at the moment. Personally I would even appreciate ordering by votes (value of the comment to the issue)

**Comment by jirihradil** [comment]
We've been waiting for this simple but crucial thing since 2014 :)

**Comment by connortsui20** [comment]
Surely it would not be _that_ difficult to do this...

**Comment by SamJaarsma** [comment]
Just copy what we see here:

![image](https://github.com/community/community/assets/122641111/cd20c6d2-73b6-4012-a548-68a83591ba4c)

**Comment by davidhunter22** [comment]
I agree with this, in fact I can't understand why the default is not newest to oldest.

**Comment by TheGiraffe3** [comment]
You can’t, but you can use the `Jump to Bottom` button and that helps.

**Comment by BrookBracewell** [comment]
Surely ordering (like on this page) by newest or oldest is an absolute must have requirement- I don't want to trawl to the bottom of a page to see the latest comments :(

**Reply by blezbrightbeam** [reply]
It's maddening for sure.

**Comment by ramimbo** [comment]
Made this as a "solution"...
https://github.com/ramimbo/github-sortout
